storage/ctab/src/main/java/org/openscience/cdk/io/MDLV2000Reader.java

```diff
@@ -1072,6 +1073,8 @@ void readPropertiesFast(final BufferedReader input, final IAtomContainer contain
                         index = readMolfileInt(line, st) - 1;
                         int value = readMolfileInt(line, st + 4);
                         SPIN_MULTIPLICITY multiplicity = SPIN_MULTIPLICITY.ofValue(value);
+                        
+                        container.getAtom(offset + index).setProperty(CDKConstants.SPIN_MULTIPLICITY, multiplicity);
```

Comment:
Don't like this.

Reply:
Or more precisely, doesn't match the API. However it is how I would have done it if we could change the past :-)